lms/djangoapps/instructor_task/tasks.py

```diff
@@ -166,3 +167,13 @@ def cohort_students(entry_id, xmodule_instance_args):
     action_name = ugettext_noop('cohorted')
     task_fn = partial(cohort_students_and_upload, xmodule_instance_args)
     return run_main_task(entry_id, task_fn, action_name)
+
+
+@task(base=BaseInstructorTask, routing_key=settings.ORA2_RESPONSES_DOWNLOAD_ROUTING_KEY)  # pylint: disable=not-callable
+def get_ora2_responses(entry_id, xmodule_instance_args):
```

Comment:
Since this exports more than just responses, I think a better name would be `export_ora2_data`.